pulsenotify/plugins/ses.py

```diff
@@ -46,6 +46,11 @@ def __init__(self):
         else:
             email_message.attach(MIMEText(status_config['message'], 'text'))
 
+        #  Set headers to create email threads
+        thread_id = '<{task_group_id}@mozilla.com>'.format(task_group_id=task_data.task_group_id)
```

Comment:
Can you avoid hardcoding `mozilla.com` here? Maybe add a new variable to dev/prod.yml?